test/unit/interfaces/express.ts

```diff
@@ -54,14 +53,14 @@ describe('expressInterface', function() {
       },
     };
     var testError = new Error('This is a test');
-    var validBoundHandler = expressInterface(client, stubbedConfig);
+    var validBoundHandler = expressInterface(client as {} as RequestHandler, stubbedConfig);
```

Comment:
Same with these (here and in other tests)... we should avoid `as {} as ...` if we can.

Reply:
My comment above applies here too.